packages/esm-home-app/src/homepage-metrics/TestDataTile.component.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Tile, Layer } from '@carbon/react';
+import styles from './home-metrics-widget.scss';
```

Comment:
You can seperate the styles or have a single file that will insinuate that its used by many files